gluster.yml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
good point, now I'm thinking about splitting `gluster.yml` into two playbooks: one just for Gluster and second for Tendrl related things on storage nodes (node-agent, performance-monitoring, gluster-integration, ceph-integration...)